src/Microsoft.Health.Dicom.Functions/Indexing/ReindexDurableFunction.Orchestration.cs

```diff
@@ -32,6 +32,7 @@ public partial class ReindexDurableFunction
         {
             EnsureArg.IsNotNull(context, nameof(context));
             logger = context.CreateReplaySafeLogger(logger);
+            await context.CallActivityAsync(nameof(FetchSchemaVersionAsync), null);
```

Comment:
We just making the call for  now without checking it?

Reply:
Will is there a way to initialize services like in web apps in Azure functions?